composer.json

```diff
@@ -20,7 +20,7 @@
         }
     ],
     "require": {
-        "php": "^5.3.9 || ^7.0",
+        "php": "^5.5 || ^7.0",
```

Comment:
Not sure if we really need the same patch level in our composer file. The min version be resolved as `5.5.9`, because of the symfony dependency.